engine/table/src/main/java/io/deephaven/engine/util/ColumnFormatting.java

```diff
@@ -74,6 +78,14 @@ static String getNumberFormatColumn(String baseColumn) {
         return baseColumn + Constants.TABLE_NUMBER_FORMAT_SUFFIX;
     }
 
+    enum DataBarFormatColumnType {
+        VALUE, AXIS, MAX, MIN, POSITIVE_COLOR, NEGATIVE_COLOR, VALUE_PLACEMENT, DIRECTION, OPACITY, MARKER, MARKER_COLOR
+    }
+
+    static String getDataBarFormatColumnName(String baseColumn, DataBarFormatColumnType prop) {
+        return baseColumn + "__" + prop.name() + Constants.TABLE_DATA_BAR_FORMAT_SUFFIX;
```

Comment:
Should lead with two `__` as well

Reply:
Do you mean like "__" + baseColumn + "__" +... ? The other formatting columns don't seem to have these two underscores at the start

<img width="366" alt="image" src="https://github.com/deephaven/deephaven-core/assets/55671206/9d0fa043-39d2-48d5-812a-5325de802801">